__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import TaskDetails from '../../../../src/components/taskDetails/index';
+
+const details = {
+  url: 'https://realdevsquad.com/tasks/6KhcLU3yr45dzjQIVm0J/details',
+  taskID: '6KhcLU3yr45dzjQIVm0J',
+  isNoteworthy: true,
+  lossRate: {
+    dinero: 0,
+    neelam: 0,
+  },
+  purpose: 'string',
+  endsOn: 1618790400,
+  title: 'test 1 for drag and drop',
+  status: 'assigned',
+  assignee: 'ankur',
+  links: ['null'],
+  dependsOn: ['null'],
+  percentCompleted: 0,
+  type: 'feature',
+  startedOn: 1618790410,
+  featureUrl: 'string',
+  completionAward: {
+    neelam: 0,
+    dinero: 110,
+  },
+};
+
+describe('TaskDetails Page', () => {
+  test('Loading text rendered when loading', () => {
+    render(<TaskDetails url={details.url} taskID={details.taskID} />);
+    const loadingElement = screen.getByText(/Loading.../i);
+    expect(loadingElement).toBeInTheDocument();
+  });
+  test('Task title is not rendered when Editing', () => {
```

Comment:
why are we hiding task title when editing?

Reply:
We are hiding as we will be showing a placeholder in it's place, which contains the existing task title as it's value 👍🏻

Reply:
Addressed it here with supporting resources
https://github.com/Real-Dev-Squad/website-status/pull/341#discussion_r1156172898